news/2020-05-14-osucatch-scoring-updates.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title: "osu!catch Scoring Updates"
+date: 2020-05-14 03:00:00 +0000
+---
+
+Thanks to the efforts of many bright-minded individuals in the community, osu!catch has just had its first Star Rating and Performance Points update in several years. Read on to discover the details!
+
+![](https://assets.ppy.sh/media/generic-header.jpg)
```

Comment:
I think there is a banner from the last star rating update that can be used, unless the file is moved to a shared folder we could use this

```suggestion
![](/wiki/shared/news/2019-02-05-new-changes-to-star-rating-performance-points/star-rating.jpg)
```